tst/EnergyPlus/unit/DXCoils.unit.cc

```diff
@@ -4489,4 +4490,246 @@ TEST_F(EnergyPlusFixture, TwoSpeedDXCoilStandardRatingsTest)
     EXPECT_EQ("10.15", RetrievePreDefTableEntry(*state, state->dataOutRptPredefined->pdchDXCoolCoilIEERIP, coolcoilTwoSpeed.Name));
 }
 
+TEST_F(EnergyPlusFixture, TwoSpeedDXCoilStandardRatings_Curve_Fix_Test)
+{
+    // Test the PR 9694 that fixes Issue 9301; will report severe error if tested in original develop branch
+    std::string const idf_objects = delimited_string({
```

Comment:
unit test.

Reply:
Would have been nice if you could produce the necessary unit test behavior with a smaller chunk of inputs, but in the grand scheme this isn't so bad.  And it works, so that's most important.